docs/UserGuide.adoc

```diff
@@ -202,19 +194,19 @@ image::demo/Open.png[]
 
 === Edit an existing note: `edit`
 
-If you want to change the `BODY` content of a note, use the `edit` command. This command *replaces* the existing `BODY` content of the currently open note, with the new `BODY` content. +
+If you want to change the `BODY` content of a note, use the `edit` command. This command *updates* the existing `BODY` content of the currently open note, with the new `BODY` content. +
```

Comment:
It's not really accurate to say that this command updates the existing body. Rather, I was thinking that we could phrase it this way:

"If you want to change the `BODY` content of the currently opened note, use the `edit` command. This command *opens* an editing window where you can update the existing `BODY` of the note.

What do you think?

Reply:
Alright, will change!